lib/OpenQA/Task/Needle/Save.pm

```diff
@@ -69,7 +70,7 @@ sub _save_needle {
     my $schema       = $app->schema;
     my $openqa_job   = $schema->resultset('Jobs')->find($args->{job_id});
     my $user         = $schema->resultset('Users')->find($args->{user_id});
-    my $needle_json  = $args->{needle_json};
+    my $needle_json  = encode_utf8($args->{needle_json});
```

Comment:
Can we add a JSON with UTF-8 characters for this to `t/16-utils-runcmd.t`?

Reply:
I added tests. But I added them in `t/ui/12-needle-editor.t` so the fronted parts are tested as well.

Reply:
Good call!